firmware/integration/rusefi_config.txt

```diff
@@ -1373,6 +1370,9 @@ uint8_t[4] unusedsw;;"units", 1, 0, -20, 100, 0
 	uint16_t mc33_t_hold_off;;"us",   1,   0.0,        0,  10000,   0
 	uint16_t mc33_t_hold_tot;;"us",   1,   0.0,        0,  10000,   0
 
+	pin_input_mode_e tcuUpshiftButtonPinMode
+	pin_input_mode_e tcuDownshiftButtonPinMode
+
 ! just a reminder that 'int' and 'float' are 4 bytes each
 	int[371] mainUnusedEnd;;"units", 1, 0, -20, 100, 0
```

Comment:
needs space for button pin modes removed from this unused chunk

Reply:
@mck1117 not really since "uint8_t alignmentFill_at_4514[2];"

Note that CI does not fail

Reply:
does something need to be done?

Reply:
@chuckwagoncomputing i think that no, you are good here

there are two other open questions
1) return meaning comment
2) return false code